controllers/vspheremachine_controller.go

```diff
@@ -128,6 +129,11 @@ func (r *VSphereMachineReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, r
 		return reconcile.Result{}, errors.Wrapf(err, "failed to create machine context")
 	}
 
+	if _, ok := vsphereMachine.Annotations[constants.MaintenanceAnnotationLabel]; ok {
+		machineContext.Logger.V(6).Info("skipping operations on vsphere machine", "reason", "annotation", "annotation-key", constants.MaintenanceAnnotationLabel)
```

Comment:
Hi @sidharthsurana,

I think we should lower the log level to `machineContext.Logger.Info` if an operation is skipped. Thoughts?

Reply:
yes, I suppose we should. Initially i thought of just using the `Logger.Info` but then just to be consistent, I went with this. IIRC there was a discussion earlier around the appropriate log level for different kinds of information. not sure if that was ever captured or not. If it is okay I will then change this to a simple `Logger.Info` instead so that it is visible whenever it is set.

BTW I close this pull request and opened a new one since my git config messed up on the new computer. Here is the new PR https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/628